pkg/networkservice/mechanisms/kernel/kernelvethpair/client.go

```diff
@@ -21,37 +21,33 @@ package kernelvethpair
 import (
 	"context"
 	"fmt"
+	"net/url"
 
 	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/pkg/errors"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/peer"
 
 	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/cls"
 
-	"github.com/networkservicemesh/sdk-vppagent/pkg/tools/netnsinode"
+	"github.com/networkservicemesh/sdk-vppagent/pkg/networkservice/vppagent"
 
 	"github.com/networkservicemesh/api/pkg/api/networkservice"
 	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
 )
 
-type kernelVethPairClient struct {
-	fileNameFromInodeNumberFunc func(string) (string, error)
-}
+type kernelVethPairClient struct{}
 
 // NewClient provides NetworkServiceClient chain elements that support the kernel Mechanism using veth pairs
 func NewClient() networkservice.NetworkServiceClient {
-	return &kernelVethPairClient{
-		fileNameFromInodeNumberFunc: netnsinode.LinuxNetNSFileName,
-	}
+	return &kernelVethPairClient{}
 }
 
 // NewTestableClient - same as NewClient, but allows provision of fileNameFromInodeNumberFunc to allow for testing
 func NewTestableClient(fileNameFromInodeNumberFunc func(string) (string, error)) networkservice.NetworkServiceClient {
```

Comment:
fileNameFromInodeNumberFunc is unused, consider removing it. Do we need NewTestableClient at all?

Reply:
Good catch.  You are correct, we don't need NewTestableClient at all anymore :)  Fixed